docs/deployment-on-heroku.rst

```diff
@@ -8,6 +8,20 @@ Run these commands to deploy the project to Heroku:
 .. code-block:: bash
 
     heroku create --buildpack https://github.com/heroku/heroku-buildpack-python
+    
+    # Setup heroku requirements
+    pip install django-heroku
```

Comment:
This could be added to the production requirements file rather than in the docs:

https://github.com/pydanny/cookiecutter-django/blob/b09d840b54f6b874a5e35c4cb59162f87d3f4cb5/%7B%7Bcookiecutter.project_slug%7D%7D/requirements/production.txt#L1-L17